### ISSUE TITLE
Any demo?

### ISSUE THREAD

**Issue by josephmiller2000** [niems/Whisper#1]
Hello is there any demo for this app? and install instruction in ubuntu or centos.

**Comment by niems** [comment]
I should have a demo up by the end of this week. I've been working on redesigning the chat ui (basically done now) and have been working on a server-side bug for the past day

**Comment by josephmiller2000** [comment]
Alright perfect :) hope i comes out in a good shape and userfriendly. Are you planning to develop it with more options and just limited ones?

**Comment by josephmiller2000** [comment]
Still waiting for the demo :)

**Comment by josephmiller2000** [comment]
I know you are updating and tweaking the UI.. but very eager to test it :) thanks.

**Comment by niems** [comment]
Hey! Yep, definitely still updating/tweaking the UI - you know how it goes, you think you're almost done with a project when you have an idea to add something else, and then another and another...

I'll send a youtube link with the current functionality by tomorrow (in the middle of adding more stuff) and will try to have the instructions done to get it running on Linux by the end of the weekend :)

**Comment by josephmiller2000** [comment]
Yeah that's true :) i"m following your project and yeah I keep noticing you are adding more features and functions. 

Yeah sure :) soon i will get that youtube video.

**Comment by niems** [comment]
Just uploaded a video of the current functionality: https://www.youtube.com/watch?v=rEfuEldPTGI

I'll work on getting install instructions for Ubuntu in a bit :)

**Comment by josephmiller2000** [comment]
Ok that looks good :).

Can we remove the add new channel permission from guest and reg users?. We dont want to end up getting 1000 new rooms in our channel. ONly give permission to create new channel for owner or admin of the chat, based on user level.

1. Does this support twitter emojis? 
2. Does this support all mobile devices like IOS and android.. and is it comfortable for users to get and view messages? though mobile devices.
3. Is there any option to integrate giphy into it? So chatters can add gifs from their server and it won't affect our server storage.

**Comment by josephmiller2000** [comment]
Is there any way to contact you? give me your email or any messaging id.. reporting things through this page is the real pain :P

**Comment by niems** [comment]
I'll have another video up in a day or so with more functionality :D In the meantime, you can message me on Twitch @ "ni3ms" with any questions/suggestions you have :)

**Comment by josephmiller2000** [comment]
did and :P no any replies.. better provide me the email id :P easy to send them.

**Comment by niems** [comment]
Weird, I didn't get anything...my email associated with Twitch is czniemann1990@gmail.com. Let me know if you try adding me again and I don't respond.

**Comment by josephmiller2000** [comment]
LOl yeah for 11 days i'm trying to connect with you through gmail :D lol i have no idea.. where is all my emails are going?
